id-policy.md

```diff
@@ -0,0 +1,231 @@
+---
+layout: doc
+title: ID Policy
+---
+
+# OBO Foundry Identifier Policy
+
+## Background
+
+This document outlines the OBO policy for identifiers and PURLs for
+OBO ontologies.
+
+In the OWL language, all elements of an ontology (including
+classes/terms, and the ontology itself) are uniquely identified by a
+[URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier). Within
+the OBO family of ontologies, we require the use of Permanent URLs
+(PURLs) with a standard base prefix.
+
+For example:
+
+ * [http://purl.obolibrary.org/obo/UBERON_0000955](http://purl.obolibrary.org/obo/UBERON_0000955) uberon class PURL for *brain*
+ * [http://purl.obolibrary.org/obo/obi.owl](http://purl.obolibrary.org/obo/obi.owl) ontology PURL for OBI
+
+Note that legacy formats such as obo-format and many bioinformatics
+databases require the use of short-form identifiers (e.g GO:0008150)
+to identify ontology terms. In this document we provide a well-defined
+mapping between these short prefixed identifiers and PURLs by
+considering the identifiers to be
+[CURIE](https://en.wikipedia.org/wiki/CURIE)s.
+
+This document addresses identifiers *only* for ontology terms, and not
+for Dbxrefs.
+
+## Design goals
+
+ - There must be a predictable, bidirectional mapping between OBO ids,
+and OBO Foundry-compliant URIs.
+ - The URIs should resolve to useful information about a term.
+ - The URIs should be designed so that they can be maintained over timeto keep pointing to useful information.
+ - Each OBO id is assigned to a only single term within the set of all OBO ontologies.
+ - There is a 1:1 mapping of OBO ids to Foundry-compliant URIs
+
+#### Format of OBO format ids
+
+See [OBO syntax](http://owlcollab.github.io/oboformat/doc/obo-syntax.html) for details.
+
+All OBO term IDs are CURIEs (prefixed identifiers) of the form `IDSpace:LocalId`
+
+Note that internally within an OBO Format document, relation IDs may
+be short form such as `part_of`, but these are mapped to CURIEs via the xref field.
+
+#### Format of Foundry-compliant URIs
+
+`FOUNDRY_OBO_URI ::= "http://purl.obolibrary.org/obo/" IDSPACE "_" LOCALID`
+
+#### Mapping of OWL ids to OBO format ids
+
+The recommended way is to use the [OBO JSON LD context](https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/context.jsonld)
+
```

Comment:
Would it be useful to add an example here?

Reply:
will do